content/zh-tw/cluster.md

```diff
@@ -0,0 +1,23 @@
+---
+title: 叢集
+status: Completed
+category: 概念
+tags: ["基礎架構", "架構", ""]
+---
+
+## 是什麼
+
+一座叢集是指一組由計算機或應用程式為實現共同目標而協同工作。
+在雲原生計算的背景之下，這個詞彙通常用於 Kubernetes。
+一個 Kubernetes 叢集是一組執行在它們自己的容器中的服務 (或工作負載)，通常在不同的機器上執行。
+所有這些 [容器化](/zh-tw/containerization/) 服務的集合，通過網路連接在一起，形成了一個叢集。
```

Comment:
original
```
The collection of all these [containerized](/containerization/) services, connected over a network, represent a cluster.
```

```suggestion
所有這些通過網絡連接的 [容器化](/zh-tw/containerization/) 服務的集合代表一座叢集。
```
In the same article, quantifiers for counting quantities remain consistent